_std/pathfinding.dm

```diff
@@ -488,7 +495,11 @@
 					else
 						return FALSE
 		if(!A.Cross(passer))
+			if(!T.pass_unstable)
+				T.passability_cache = FALSE
```

Comment:
You're setting it to false here but checking against null above, intended?

Reply:
Intentional, this is the cache write. Because Cross failed, it will write to the cache that it is not passable.